doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -766,58 +726,81 @@ the conversion in hypotheses :n:`{+ @ident}`.
          pattern (0 = 0).
          fold not.
 
-   .. tacv:: fold {+ @term}
+.. tacn:: pattern {+, @pattern_occs } {? @occurrences }
 
-      Equivalent to :n:`fold @term ; ... ; fold @term`.
+   .. todo PR: how do multiple pattern_occs work with @occurrences??
```

Comment:
I believe the `at` clause of `@occurrences` is not supported?

Reply:
If it were only that simple.  It seems to work like `unfold`.

```
Goal 2+3+4 = 0.
pattern 2,3 in |- * at 1.  (* "at" clause not supported in presence of an occurrence clause. *)

pattern 2 in |- * at 1.  (* (fun n : nat => n + 3 + 4 = 0) 2 *)